src/main/webapp/i18n/de/modelingExercise.json

```diff
@@ -7,6 +7,8 @@
                 "editLabel": "Modellierungsaufgabe bearbeiten",
                 "importLabel": "Modellierungsaufgabe importieren"
             },
+            "create": "Modellierung erstellen",
+            "import": "Modellierung importieren",
```

Comment:
I think it should contain "exercise" and we should not duplicate translations keys